pkg/authz/authsvc/callback.go

```diff
@@ -125,3 +124,37 @@ func (c *callback) makeRequest(ctx context.Context, urlStr string, params map[st
 
 	return nil
 }
+
+func NewCalloutClient(authCalloutTLSVerification bool, authCalloutCert, authCalloutKey, authCalloutCACert string, authCalloutTimeout time.Duration) (*http.Client, error) {
```

Comment:
Rename params to `tls bool, certPath, keyPath, caPath string, timeout time.Duration`.